nxdlTypes.xsd

```diff
@@ -113,6 +113,16 @@
 		<xs:restriction base="xs:string" />
 	</xs:simpleType>
 	
+	<xs:simpleType name="NX_COUNT">
+		<xs:annotation>
+			<xs:documentation>
+				units of quantity of item(s) such as number of photons,
+				neutrons, or other counting events
+			</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="nxdl:NX_INT" />
```

Comment:
Not string as all units attributes are strings?

Reply:
Right!  What about these?  Same correction?

https://github.com/nexusformat/definitions/blob/99fb0e0ee50e57ed049566ff16f3159a083ddf3c/nxdlTypes.xsd#L259-L268

https://github.com/nexusformat/definitions/blob/99fb0e0ee50e57ed049566ff16f3159a083ddf3c/nxdlTypes.xsd#L290-L297

https://github.com/nexusformat/definitions/blob/99fb0e0ee50e57ed049566ff16f3159a083ddf3c/nxdlTypes.xsd#L343-L352

Reply:
created as new issue